ModuleConfig.cfc

```diff
@@ -4,22 +4,205 @@ component {
     this.author = 'John Berquist';
     this.webURL = 'https://github.com/jcberquist/stripe-cfml';
     this.description = 'This module will provide you with connectivity to the Stripe API for any ColdFusion (CFML) application.';
+    this.entryPoint = 'stripe-cfml';
+    this.modelNamespace = 'stripe-cfml';
+    this.cfmapping = 'stripe-cfml';
```

Comment:
I am curious about the name `stripe-cfml` on all three lines here - what I currently have referenced in the readme for module settings is `stripecfml` (no dash). That is how the slug is named in the `box.json` as well, and I was under the impression that the module would be named after the install directory (which would match the slug) by WireBox. Is there something I am missing here?

Reply:
These can technically be different, but this was just me matching the GitHub repo instead of the ForgeBox slug.  I'll update it to `stripecfml`.